data/release/man/c.1

```diff
@@ -443,11 +443,15 @@ State file that captures the terminal screen and store it as json.
 .SH REFERENCE
 .INDENT 0.0
 .IP 1. 3
-Command Line Assistant source code: \X'tty: link https://github.com/rhel-lightspeed/command-line-assistant'\fI\%https://github.com/rhel\-lightspeed/command\-line\-assistant\fP\X'tty: link'
+Command Line Assistant source code: <\X'tty: link https://github.com/rhel-lightspeed/command-line-assistant'\fI\%https://github.com/rhel\-lightspeed/command\-line\-assistant\fP\X'tty: link'>
 .UNINDENT
 .SH BUGS
 .sp
-Please send bug reports to our bug tracker, see \X'tty: link https://issues.redhat.com/browse/RSPEED'\fI\%https://issues.redhat.com/browse/RSPEED\fP\X'tty: link'
+To submit bug reports, please use the following link:
+<\X'tty: link https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12332745&priority=10200&issuetype=1&components=12410340'\fI\%https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12332745&priority=10200&issuetype=1&components=12410340\fP\X'tty: link'>
+.sp
+In case it is a feature request, please use the following link:
+<\X'tty: link https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12332745&priority=10200&issuetype=2&components=12410340'\fI\%https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12332745&priority=10200&issuetype=2&components=12410340\fP\X'tty: link'>
```

Comment:
This link still points to the Feature Request type without the predefined component set